lisa/features/gpu.py

```diff
@@ -177,7 +172,7 @@ def get_supported_driver(self) -> List[ComputeSDK]:
         raise NotImplementedError()
 
     def _initialize(self, *args: Any, **kwargs: Any) -> None:
-        self.gpu_vendor: Set[str] = set()
+        self.gpu_vendor: List[str] = ["nvidia"]
```

Comment:
What happened, if the vendor is not nvidia?

Reply:
from current logic, I found all of them use nvidia and assigned in the code, maybe I can move it to azure feature.

Reply:
Can it be checked from code?

Reply:
updated, now use whether there is any module used for nvidia pci devices, if no driver installed, the used module is empty.